manifests/install.pp

```diff
@@ -40,10 +40,16 @@
     }
   }
 
-  $file = "atlassian-${jira::product}-${jira::version}.${jira::format}"
+  # https://www.atlassian.com/software/jira/downloads/binary/atlassian-jira-software-7.0.0-jira-7.0.0-x64.bin
+
+  if ($jira::version[0] == '7' ){
+    $file = "atlassian-jira-software-7.0.0-${jira::product}-${jira::version}${jira::format}"
+  }else {
+    $file = "atlassian-${jira::product}-${jira::version}${jira::format}"
```

Comment:
There is more than 2 options..

JIRA Core:
atlassian-jira-core-7.0.0.tar.gz
JIRA Software (JIRA + Agile):
atlassian-jira-software-7.0.0-jira-7.0.0.tar.gz
JIRA Service Desk (JIRA + Service Desk):
atlassian-servicedesk-3.0.0-jira-7.0.0.tar.gz